CONTRIBUTING.md

```diff
@@ -4,11 +4,11 @@ Thanks for your interest in Noda Time. We appreciate all kinds of contributions,
 
 ## Basic Requirements
 
-If you want to contribute to the codebase, you're going to need a text editor or IDE. We recommend [Visual Studio Community 2017](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx) (Windows) or [Visual Studio Code](https://code.visualstudio.com/) (Windows/Linux/macOS).
+If you want to contribute to the codebase, you're going to need a text editor or IDE. We recommend [Visual Studio Community](https://visualstudio.microsoft.com/downloads/) (Windows), [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/), or [Visual Studio Code](https://code.visualstudio.com/) (Windows/Linux/macOS).
 
-You're also going to need .NET Core SDK installed - you can download it from [here.](https://www.microsoft.com/net/download/) (For Noda Time 3.0, we're using C# 8, which as of 2018-12-31 means using a preview of .NET Core.)
+You're also going to need .NET Core SDK installed - you can download it from [here](https://dotnet.microsoft.com/download).
```

Comment:
Updated the link. Removed the preview note. I think it's outdated now.

Reply:
Yup, that's fine.